src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -70,7 +70,45 @@ public void storeMessage(Message message) {
 
         Message message = new Message(id, user, text, timestamp);
         messages.add(message);
-      } catch (Exception e) {
+      } 
```

Comment:
Small nit-pick, but usually, the catch keyword is usually on the same line as the closing brace for the try block:
```
try {
  ...
} catch (Exception e) {
  ...
}
```

Reply:
Ok got it!